CODE_OF_CONDUCT.md

```diff
@@ -1 +1,90 @@
-TBD
+# FaithTech LABS Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+FaithTech LABS pledge to making participation in
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behaviour that contributes to creating a positive environment
+include:
+
+* Being friendly and patient
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Trying to understand why we disagree
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behaviour by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+FaithTech is responsible for clarifying the standards of acceptable
+behaviour and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behaviour.
+
+We have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any individual for other behaviours that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Our Faith
+
+> For it is by grace you have been saved, through faith - and this is not from yourselves, it is the gift of God - not by works, so that no one can boast. For we are God’s handiwork, created in Christ Jesus to do good works, which God prepared in advance for us to do.
+> [Ephesians 2:8-10](https://www.bible.com/bible/111/EPH.2.8-10)
+
+We will always be grounded in our faith and beliefs. Because of our beliefs we will always be open and loving:
```

Comment:
This is good! Maybe a slight addition to help some people just skimming through make the connection?

`FaithTech is a faith based organization, and as such we will always be...`

Reply:
Updating to `We, FaithTech,` I want to continue writing the document from the first person plural. Since this is a document meant to represent the whole community I felt it was important to keep it this way.